app/controllers/record_controller.rb

```diff
@@ -0,0 +1,23 @@
+class RecordController < ApplicationController
+  rescue_from EBSCO::EDS::BadRequest, with: proc {
+    raise ActionController::RoutingError, 'Record not found'
+  }
+
+  def record
+    record_source = params[:db_source]
+    record_an = params[:an]
+    return redirect_to root_url unless record_source && record_an
+    session = EBSCO::EDS::Session.new(user: ENV['EDS_USER_ID'],
```

Comment:
'As they expect' is unclear, since this line is copy-pasted from their own documentation. (https://github.com/ebsco/edsapi-ruby/wiki/Quick-Start, modulo codeclimate tweaks to syntax)